src/Native/Unix/configure.cmake

```diff
@@ -362,6 +362,34 @@ check_prototype_definition(
 
 check_cxx_source_compiles(
     "
+    #include <stdlib.h>
+
+    int main()
+    {
+        char* path = strdup(\"abc\");
+        int ret = mkstemps(path, 3);
```

Comment:
OK, I merged the return statement - I think that's as simple as I can make that check at this point.